src/session/sessionCookieStore.ts

```diff
@@ -1,39 +1,27 @@
-import { simpleSessionCookieStore } from './app-session-cookie-store'
-import { shouldRefresh } from './shouldRefresh/shouldRefresh'
-import { refreshAppSession } from './refreshAppSession/refreshAppSession'
-import { refreshToken } from '../storyblok-auth-api/refreshToken'
-import { AppSessionCookieStoreFactory } from './types'
-import { AppSessionStore } from './types'
+import { AppSessionCookieStoreFactory, AppSessionStore } from './types'
+import { getAllSessions, getSession, putSession, removeSession } from './crud'
+import { refreshStoredAppSession } from './refreshStoredAppSession'
+import { GetCookie, getCookie, SetCookie, setCookie } from '../utils'
 
 export const sessionCookieStore: AppSessionCookieStoreFactory =
   (params) =>
   (requestParams): AppSessionStore => {
-    const store = simpleSessionCookieStore(params)(requestParams)
+    const getNodeCookie: GetCookie = (name) =>
+      getCookie(requestParams.req, name)
+    const setNodeCookie: SetCookie = (name, value) =>
+      setCookie(requestParams.res, name, value)
```

Comment:
Could you explain to me about these ***NodeCookie functions? What's their purpose?

Reply:
`refreshStoredAppSession`, `getAllSessions`, `putSession`, `removeSession` all expect a getters and setters functions as arguments that do not rely on `http.IncomingMessage` or `http.ServerResponse`. So `getNodeCookie` and `setNodeCookie` here just close over the `req` and `res` objects.

The names `getCookie` and `setCookie` were already taken, hence the names...

Btw, the naming is a bit confusing, so I aliased swapped the names here: https://github.com/storyblok/app-extension-auth/pull/7/commits/0b2c361aec581ea7c61ce84b7f0afc085c208505#diff-5f3663d1490789fb11f664fc029e976ad270b799d0f641818b837bddf1d4b6b0R4-R17

Reply:
Thanks for the explanation. I'm getting close!